web/karma.conf.js

```diff
@@ -0,0 +1,69 @@
+// Karma configuration
```

Comment:
@DylanWalseth You got an recs for learning how to robustly test API? I asked someone at work for advice and will meet with this person in the future.